crates/nu-command/src/formats/to/toml.rs

```diff
@@ -49,9 +49,7 @@ fn helper(engine_state: &EngineState, v: &Value) -> Result<toml::Value, ShellErr
         Value::Int { val, .. } => toml::Value::Integer(*val),
         Value::Filesize { val, .. } => toml::Value::Integer(*val),
         Value::Duration { val, .. } => toml::Value::String(val.to_string()),
-        Value::Date { val, .. } => {
-            toml::Value::String(val.to_rfc3339_opts(SecondsFormat::AutoSi, false))
```

Comment:
Seems like you could've just used this to_rfc3339_opts function and removed the surrounding quotes to achieve the same result. I don't see any difference in the format other than the quotes.

Reply:
@fdncred I don't see a way to remove the quotes, because everything is encoded using `toml`'s machinery.  `to_toml` takes a `toml::Value`, which has it's own encoding rules and requires the `toml::Value::Datetime` variant for bare dates.

Reply:
And `toml` doesn't provide any useful conversion traits, be it for chrono or strings, so I was stuck converting everything to their ad-hoc format by hand

Reply:
Ah, ok.